### PR TITLE
working hello world, docker file for postgres DB

### PR DESCRIPTION
**Pull request by CamPlume1**

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      test:
+        [
+          "CMD",
+          "pg_isready",
+          "-q",
+          "-d",
+          "postgres",
+          "-U",
+          "user"
+        ]
```
Comment (alexangione419) [inline docker-compose.yaml:15]: $${\huge\color{limegreen}\textbf{[+1]}}$$ good

File: docker-compose.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+services:
+  db:
+    image: postgres:14.1-alpine
+    restart: always
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "pg_isready",
+          "-q",
```
Comment (kennybc) [inline docker-compose.yaml:10]: test

File: docker-compose.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+          "-d",
+          "postgres",
+          "-U",
+          "user"
```
Comment (kennybc) [inline docker-compose.yaml:14]: $${\huge\color{WildStrawberry}\textbf{[-7]}}$$ this was bad

Comment (kennybc) [inline docker-compose.yaml:14]: $${\huge\color{limegreen}\textbf{[+6]}}$$ this was good

File: docker-compose.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+services:
```
[the remark this answers is not in the document]
Comment (kennybc) [reply docker-compose.yaml:1]: regrade this

[the remark this answers is not in the document]
Comment (kennybc) [reply docker-compose.yaml:1]: regrade this

[the remark this answers is not in the document]
Comment (kennybc) [reply docker-compose.yaml:1]: test

**Comment by kennybc** [comment]
test

File: docker-compose.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  db:
+    image: postgres:14.1-alpine
+    restart: always
+    healthcheck:
```
Comment (alexangione419) [inline docker-compose.yaml:5]: $${\huge\color{limegreen}\textbf{[+2]}}$$ liujdbfljsdba

File: docker-compose.yaml (hunk 4 above)
Comment (kennybc) [inline docker-compose.yaml:1]: $${\huge\color{limegreen}\textbf{[+3]}}$$ test

Comment (kennybc) [inline docker-compose.yaml:1]: $${\huge\color{limegreen}\textbf{[+10]}}$$ Tracks running distances accurately

File: backend/internal/server/server.go [hunk 6]
```diff
@@ -2,8 +2,9 @@ package server
 
 import (
 	"github.com/CamPlume1/khoury-classroom/internal/errs"
-	"github.com/CamPlume1/khoury-classroom/internal/types"
+	hello "github.com/CamPlume1/khoury-classroom/internal/handlers"
 	"github.com/CamPlume1/khoury-classroom/internal/middleware"
+	"github.com/CamPlume1/khoury-classroom/internal/types"
 	go_json "github.com/goccy/go-json"
 	"github.com/gofiber/fiber/v2"
 	"github.com/gofiber/fiber/v2/middleware/compress"
```
Comment (sebytremblay) [inline backend/internal/server/server.go:10]: $${\huge\color{limegreen}\textbf{[+10]}}$$ Tracks running distances accurately

File: docker-compose.yaml [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+      - POSTGRES_USER=user
+      - POSTGRES_DB=classroom
+      - POSTGRES_PASSWORD=pwd
+    ports:
+      - '5434:5432'
+    volumes:
+      - ./database/migrations:/docker-entrypoint-initdb.d/
+volumes:
+  db:
+    driver: local
```
Comment (sebytremblay) [inline docker-compose.yaml:29]: $${\huge\color{limegreen}\textbf{[+10]}}$$ Tracks running distances accurately

File: docker-compose.yaml [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+        [
+          "CMD",
+          "pg_isready",
+          "-q",
+          "-d",
+          "postgres",
+          "-U",
+          "user"
+        ]
+      timeout: 45s
```
Comment (sebytremblay) [inline docker-compose.yaml:16]: $${\huge\color{limegreen}\textbf{[+10]}}$$ Tracks running distances accurately

File: docker-compose.yaml [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+      interval: 10s
```
Comment (sebytremblay) [inline docker-compose.yaml:17]: $${\huge\color{limegreen}\textbf{[+10]}}$$ Tracks running distances accurately

Comment (sebytremblay) [inline docker-compose.yaml:17]: $${\huge\color{limegreen}\textbf{[+10]}}$$ Records chocolate consumption correctly

Comment (sebytremblay) [inline docker-compose.yaml:17]: $${\huge\color{limegreen}\textbf{[+10]}}$$ Code is well-documented and clean

Comment (sebytremblay) [inline docker-compose.yaml:17]: $${\huge\color{limegreen}\textbf{[+10]}}$$ Tracks running distances accurately

Comment (sebytremblay) [inline docker-compose.yaml:17]: $${\huge\color{limegreen}\textbf{[+10]}}$$ Records chocolate consumption correctly

Comment (sebytremblay) [inline docker-compose.yaml:17]: $${\huge\color{limegreen}\textbf{[+10]}}$$ Code is well-documented and clean

File: docker-compose.yaml [hunk 10]
```diff
@@ -0,0 +1,29 @@
+services:
+  db:
+    image: postgres:14.1-alpine
```
Comment (sebytremblay) [inline docker-compose.yaml:3]: $${\huge\color{limegreen}\textbf{[+10]}}$$ Tracks running distances accurately

Comment (sebytremblay) [inline docker-compose.yaml:3]: $${\huge\color{limegreen}\textbf{[+10]}}$$ Records chocolate consumption correctly

Comment (sebytremblay) [inline docker-compose.yaml:3]: $${\huge\color{limegreen}\textbf{[+10]}}$$ Code is well-documented and clean

**Pull request merged**